### DISCUSSION TITLE
use cli to generate css but transformers unable to use @apply

### DISCUSSION THREAD

**Discussion by LanJamRom** [unocss/unocss#1313] [Q&A]
[demo](https://github.com/LanJamRom/umi-unocss-bug.git)
this project use [@umijs/plugins](https://github.com/umijs/umi/blob/master/packages/plugins/src/unocss.ts) generate uno.css
it just doesn't support this mode?

**Comment by zyyv** [comment]
Yes, [@umijs/plugins](https://github.com/umijs/umi/blob/master/packages/plugins/src/unocss.ts) use unocss cli start server, but cli not support transformer yet. 
Related to #1369
